utils/+u19_dj_utils/getParametersFromQuery.m

```diff
@@ -0,0 +1,53 @@
+function [params] = getParametersFromQuery(query)
```

Comment:
put into +imaging/+utils might work after merging with meso

Reply:
ok, I will change some u19_dj_utils general functions closer to the context they are being used. In particular I will change this one to +imaging/+utils